frontend/package.json

```diff
@@ -8,7 +8,8 @@
     "build": "tsc && vite build",
     "lint": "biome check --apply-unsafe --no-errors-on-unmatched --files-ignore-unknown=true ./",
     "preview": "vite preview",
-    "generate-client": "openapi-ts --input ./openapi.json --output ./src/client --client axios --exportSchemas true && biome format --write ./src/client"
+    "generate-client": "openapi-ts --input ./openapi.json --output ./src/client --client axios --exportSchemas true && biome format --write ./src/client",
```

Comment:
what is this?

Reply:
this automate the process of generating a client library from an OpenAPI specification, it also automatically formats it adhering to guidelines